### ISSUE TITLE
Call for Public Tests

### ISSUE THREAD

**Issue by alessiogambi** [IMC-UAS-Krems/2024-DBST2VO-Public#1]
Please add public tests to this thread. Each message you post must contain exactly ONE public test.
Add a small description (one/two sentences)
Add some sample code with comments
Specify any assumption (e.g., connecting as user X, the db must contain these data, the test can be run using SQLite, etc.)

**Comment by alessiogambi** [comment]
#### Description: Check that admins cannot insert the same traits_user twice
#### Sample Code:
```
import pytest

def test_no_user_duplicate(traits_on_sqlite_empty):
   user_email = "a@valid-email.com"
   user_details = {"name" : "foo"} # Assume this is Ok. 
   user_different_details = {"name" : "bar"} # Assume this is Ok. This
   # Insert the user
   traits_on_sqlite.add_user(user, user_details)
   # Try to insert the same user again with different details. Expect a ValueError
   with pytest.raises(ValueError) as e_info:
       traits_on_sqlite.add_user(user, user_different_details)
```

#### Assumptions:
- the fixture `traits` is the implementation and can establish a connection to a sqlite local db
- the local DB is empty or at least does not contain already the user to add

**Comment by alessiogambi** [comment]
I added a bunch of public tests in [#3cf11e119bdd10c3](https://github.com/IMC-UAS-Krems/2024-DBST2VO-Public/commit/3a36501d94d917b9da2e50bd8a6add5fcda65044). Those tests cover all the methods, but are not currently working because we need to fix #3

**Comment by alessiogambi** [comment]
At this point, I do not think anyone will contribute more public tests. So I close the issue. In case you have a change of mind, reopen the issue and tag me.
